PlayCover/Views/CachedAsyncImageWrapper.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  CachedAsyncImageWrapper.swift
+//  PlayCover
+//
+//  Created by TheMoonThatRises on 1/28/25.
+//
+
+import CachedAsyncImage
+import SwiftUI
+
+struct CachedAsyncImageWrapper: View {
+
+    let url: String?
+    let placeholder: ((String) -> any View)?
```

Comment:
isn't the whole reason of creating this wrapper to make sure there is an image url to grab and show?
making it optional and also another view is not good approach from my point of view.
maybe adding a placeholder image to assets and load that as an image a better way to handle it instead.

Reply:
The reason for the wrapper is to handle empty or nil url inputs, as the `CachedAsyncImage` library cannot handle it and will return an error into the console when attempting to resolve a nil or empty string.

Reply:
Show a ProgressView instead of creating an empty input.

Reply:
It doesn't create an empty view, it uses the `placeholder` provided to the original `CachedAsyncImage` if it is unable to resolve the url. This wrapper should provide the same behavior as before, but this wrapper simply makes it so the package doesn't log an error and provide undefined behavior if the url is nil or empty.

Reply:
if it doesn't change behaviour, then lgtm